folly/container/test/SparseByteSetTest.cpp

```diff
@@ -66,3 +66,15 @@ TEST_F(SparseByteSetTest, each_random) {
     EXPECT_TRUE(s.contains(c));
   }
 }
+
+TEST_F(SparseByteSetTest, clear) {
+  for (auto c = lims::min(); c < lims::max(); ++c) {
+    EXPECT_TRUE(s.add(c));
+  }
+  s.clear();
+  for (auto c = lims::min(); c < lims::max(); ++c) {
```

Comment:
Swapping this to count down from max() to min() should expose the issue I'm talking about above.

Reply:
This test are passed. Quick example for you: https://godbolt.org/z/vdEvhGWE4

Reply:
Ah, I missed an important part of how this actually works. This should be fine then :)